content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
+
+Only single sign-on user accounts are allowed. Service accounts, such as deployer credentials, are permissible with the condition that they are scoped to a particular space with limited access.
```

Comment:
On the first sentence: Will users know what a single sign-on user account is, and do they have any control over it? I think we should reframe this from the user's perspective... For example "We only provision access to this environment for agencies prepared to actively manage their user accounts, eg via a SAML identity provider. If you don't see your agency listed on [the login page] you may wish to check with your agency to see if they can provide this capability; we will set up integration upon request."